test/controllers/images_controller_test.rb

```diff
@@ -102,4 +102,21 @@ class ImagesControllerTest < ActionController::TestCase
 
     assert_equal true, JSON.parse(@response.body)['downvoted']
   end
+
+  test "should be able to get image comments" do
+    image = Image.find images(:one).id
+    post :comments, {id: image.id}
+
+    assert_equal "First post!!!1!", JSON.parse(@response.body)[0]['text']
+  end
+
+  test "should be able to post comment" do
+    image = Image.find images(:one).id
+    og_comment_count = image.comments.count
+    post :add_comment, {id: image.id, text: "whooaaaa"}
+
+    image.reload
+    assert_equal og_comment_count+1, image.comments.count
+    assert_equal "whooaaaa", image.comments.last.text
+  end
```

Comment:
As good form, it's better to toss the comment text into a variable on insertion, and then compare the value to the variable on `assert_equal`. If someone needs to change the body for whatever reason, it keeps them from screwing up the test.